fdbserver/workloads/SkewedReadWrite.actor.cpp

```diff
@@ -0,0 +1,778 @@
+/*
+ * ReadWrite.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <boost/lexical_cast.hpp>
+#include <utility>
+#include <vector>
+
+#include "fdbrpc/ContinuousSample.h"
+#include "fdbclient/NativeAPI.actor.h"
+#include "fdbserver/TesterInterface.actor.h"
+#include "fdbserver/WorkerInterface.actor.h"
+#include "fdbserver/workloads/workloads.actor.h"
+#include "fdbserver/workloads/BulkSetup.actor.h"
+#include "fdbclient/ReadYourWrites.h"
+#include "flow/TDMetric.actor.h"
+#include "fdbclient/RunTransaction.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+const int sampleSize = 10000;
+DESCR struct TransactionSuccessMetric {
+	int64_t totalLatency; // ns
+	int64_t startLatency; // ns
+	int64_t commitLatency; // ns
+	int64_t retries; // count
+};
+
+DESCR struct TransactionFailureMetric {
+	int64_t startLatency; // ns
+	int64_t errorCode; // flow error code
+};
+
+DESCR struct ReadMetric {
+	int64_t readLatency; // ns
+};
```

Comment:
As much as possible, the code shared between `SkewedReadWrite` should be moved into its own separate file (e.g. `ReadWrite.h`) so that it doesn't have to be duplicated in these two workloads.